react/features/base/components/common/Button.tsx

```diff
@@ -0,0 +1,199 @@
+            border: 0,
+            ...withPixelLineHeight(theme.typography.bodyShortBold),
+            transition: 'background .2s',
+            cursor: 'pointer',
+
+            '&:hover': {
+                backgroundColor: theme.palette.action01Hover
+            },
+
+            '&:active': {
+                backgroundColor: theme.palette.action01Active
+            },
+
+            '&:focus': {
+                outline: 0,
+                boxShadow: `0px 0px 0px 2px ${theme.palette.focus01}`
+            },
+
+            '& svg': {
+                fill: theme.palette.icon01
+            }
+        },
+
+        primary: {},
+
+        secondary: {
+            backgroundColor: theme.palette.action02,
+            color: theme.palette.text04,
+
+            '&:hover': {
+                backgroundColor: theme.palette.action02Hover
+            },
+
+            '&:active': {
+                backgroundColor: theme.palette.action02Active
+            },
+
+            '& svg': {
+                fill: theme.palette.icon04
+            }
+        },
+
+        tertiary: {
+            backgroundColor: theme.palette.action03,
+
+            '&:hover': {
+                backgroundColor: theme.palette.action03Hover
+            },
+
+            '&:active': {
+                backgroundColor: theme.palette.action03Active
+            }
+        },
+
+        destructive: {
+            backgroundColor: theme.palette.actionDanger,
+
+            '&:hover': {
+                backgroundColor: theme.palette.actionDangerHover
+            },
+
+            '&:active': {
+                backgroundColor: theme.palette.actionDangerActive
+            }
+        },
+
+        disabled: {
+            backgroundColor: theme.palette.disabled01,
+            color: theme.palette.text03,
+
+            '&:hover': {
+                backgroundColor: theme.palette.disabled01,
+                color: theme.palette.text03
+            },
+
+            '&:active': {
+                backgroundColor: theme.palette.disabled01,
+                color: theme.palette.text03
+            },
+
+            '& svg': {
+                fill: theme.palette.icon03
+            }
+        },
+
+        iconButton: {
+            padding: '10px'
+        },
+
+        textWithIcon: {
+            marginLeft: `${theme.spacing(2)}px`
+        },
+
+        small: {
+            padding: '8px 16px',
+            ...withPixelLineHeight(theme.typography.labelBold),
+
+            '&.iconButton': {
+                padding: '6px'
+            }
+        },
+
+        medium: {},
+
+        large: {
+            padding: '13px 16px',
+            ...withPixelLineHeight(theme.typography.bodyShortBoldLarge),
+
+            '&.iconButton': {
+                padding: '14px'
+            }
+        },
+
+        fullWidth: {
+            width: '100%'
+        }
+    };
+});
+
+const Button = ({
+    accessibilityLabel,
+    disabled,
+    fullWidth,
```

Comment:
I think we can make the button have a full width by default and not rely on this prop.

Reply:
On web most buttons will not be full width, so I think it's better to have it like this

Reply:
This means we can make it a prop with a false default value.

Reply:
It is a prop with a falsy default value

Reply:
You def right.